### ISSUE TITLE
How to improve the low qulity image detail?

### ISSUE THREAD

**Issue by JunCS1** [junyanz/pytorch-CycleGAN-and-pix2pix#1597]
Hi, I'm modifying the current repository 2d cyclegan into 3d cyclegan in order to use on medical image transition for different tracers. However, I tried various different configurations but most of the inferenced image I got are lack in details. Could you provide me any tips to improve the details? I'm trying with larger --ngf, --ndf and larger patch size to see if the network can learn more details. Also I'm adding data augmentation like horizontal or vertail flip, -30 to 30 degree random rotation on x, y and z. 

Image 1: Real A( Real Image of Tracer A as the input of Inference)
<img width="703" alt="Screenshot 2023-09-18 at 2 27 41 PM" src="https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/assets/87685037/940530e8-bc2f-4a4f-9f20-d015abe47fa8">

Image 2: Real B(This is not the ground truth of Image 3, but it's the real image of Tracer B, the goal of Image I want)

<img width="750" alt="Screenshot 2023-09-18 at 2 30 16 PM" src="https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/assets/87685037/b2f60b7a-04df-47c6-a0a8-fc79e5b65a9f">

Image 3: Fake B I got from inference from Image 1 (Result from epoch 180, Learning rate is keeping at 0.0002, learning rate will gradually decay to zero from epoch 200 to 400), 

<img width="708" alt="Screenshot 2023-09-18 at 2 28 20 PM" src="https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/assets/87685037/1746597e-aa30-4a0b-8d43-291fcd8f83dd">

**Comment by lypliuyouping** [comment]
It is recommended to crop the image as there is too much interference from the black background
